pyquil/api/_benchmark.py

```diff
@@ -102,10 +112,17 @@ def generate_rb_sequence(self, depth, gateset, seed=None):
         gateset_as_program = address_qubits(sum(gateset, Program()))
         qubits = len(gateset_as_program.get_qubits())
         gateset_for_api = gateset_as_program.out().splitlines()
+        if interleaver:
+            assert(isinstance(interleaver, Program))
+            interleaver = interleaver.out()
 
         depth = int(depth)  # needs to be jsonable, no np.int64 please!
 
-        payload = RandomizedBenchmarkingRequest(depth=depth, qubits=qubits, gateset=gateset_for_api, seed=seed)
+        payload = RandomizedBenchmarkingRequest(depth=depth,
+                                                qubits=qubits,
+                                                gateset=gateset_for_api,
+                                                seed=seed,
+                                                interleaver=interleaver)
```

Comment:
how does rpcq work? If I upgrade my pyquil but leave `interleaver=None`, will rpcq accept it or throw an error? if the latter, do we need to bump `requirements.txt`

Reply:
```
rpcq/messages.py:505: UserWarning: Message RandomizedBenchmarkingRequest ignoring unexpected keyword arguments: interleaver.
```
It whines, but it doesn't die.

Reply:
is that what youre asking @mpharrigan? if you upgrade pyquil but not the reqs?

Reply:
yes thanks